### PR TITLE
distribution metrics for cloud run and azure container apps

### PR DESCRIPTION
**Pull request by cswatt**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/serverless/azure_container_apps/_index.md [hunk 1]
```diff
@@ -110,9 +110,13 @@ See [Tracing Ruby Applications][1] for detailed instructions. [Sample code for a
 {{< /programming-lang-wrapper >}}
 
 ### Custom metrics
-You can submit custom metrics using a [DogStatsd client][4].
+You can submit custom metrics using a [DogStatsD client][4].
 
-**Note**: Only `DISTRIBUTION` metrics should be used.
+For monitoring Cloud Run and other serverless applications, use [distribution][7] metrics.
```
Comment (buraizu) [inline content/en/serverless/azure_container_apps/_index.md:115]:
```suggestion
For monitoring Azure App Service and other serverless applications, use [distribution][8] metrics.
```

File: content/en/serverless/azure_container_apps/_index.md [hunk 2]
```diff
@@ -110,9 +110,13 @@ See [Tracing Ruby Applications][1] for detailed instructions. [Sample code for a
...
+
+Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][8]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
```
Comment (buraizu) [inline content/en/serverless/azure_container_apps/_index.md:117]:
```suggestion
Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][9]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
```

File: content/en/serverless/azure_container_apps/_index.md [hunk 3]
```diff
@@ -110,9 +110,13 @@ See [Tracing Ruby Applications][1] for detailed instructions. [Sample code for a
...
+
+For more information, see [Serverless Custom Metrics - Understanding distribution metrics][6].
```
Comment (buraizu) [inline content/en/serverless/azure_container_apps/_index.md:119]:
```suggestion
For more information, see [Serverless Custom Metrics - Understanding distribution metrics][7].
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, it looks like some links are being mis-referenced, and I also added a suggestion to update the example on the Azure page to mention the Azure serverless offering.

File: content/en/serverless/azure_container_apps/_index.md [hunk 4]
```diff
@@ -143,3 +147,6 @@ You can use the [Azure integration][1] to collect logs. Alternatively, you can s
 [4]: /metrics/custom_metrics/dogstatsd_metrics_submission/
 [5]: /getting_started/site/
 [6]: /getting_started/tagging/unified_service_tagging/
+[7]: /serverless/custom_metrics/#understanding-distribution-metrics
```
Comment (tianchu) [inline content/en/serverless/azure_container_apps/_index.md:150]: The rest of the content of that page is about Lambda, so I'm worried that linking user from a azure container apps doc to a lambda doc could cause confusion. I think we can probably leave out the link or we should refactor that section from the lambda doc to a standalone guide.

**Review by tianchu — APPROVED** [review]
Only had a minor comment, otherwise looks good to me.

**Review by buraizu — APPROVED** [review]

**Pull request merged**